avocado/plugins/run.py

```diff
@@ -162,9 +162,11 @@ def run(self, args):
             sys.exit(exit_codes.AVOCADO_FAIL)
         job_instance = job.Job(args)
         job_run = job_instance.run()
-        result_dispatcher = ResultDispatcher()
-        if result_dispatcher.extensions:
-            result_dispatcher.map_method('render',
-                                         job_instance.result,
-                                         job_instance)
+
+        if job_instance.status != 'RUNNING':
```

Comment:
(Consider an improved human UI plugin. You do want to receive all the information, not only when the system finishes properly)